Italian/main/TeleService.apk/res/values-it/strings.xml

```diff
@@ -1006,8 +1006,8 @@ Sarà usata al suo posto la rete preferita.</string>
     <string name="auto_answer_scenario_title">Condizione per risposta automatica</string>
     <string name="auto_answer_scenario_enteries_always">Sempre</string>
     <string name="auto_answer_scenario_enteries_headset_only">Solo auricolari</string>
-    <string name="auto_answer_scenario_enteries_bluetooth_only">Solo bluetooth</string>
-    <string name="auto_answer_scenario_enteries_headset_bluetooth">Solo auricolari o bluetooth</string>
+    <string name="auto_answer_scenario_enteries_bluetooth_only">Solo Bluetooth</string>
+    <string name="auto_answer_scenario_enteries_headset_bluetooth">Solo auricolari o Bluetooth</string>
```

Comment:
Oddio a me sembra un po' azzardata come mossa, e ci vedo più contro che pro.
Potrei capire il cambiamento dettato da esigenze estetiche, ma a parte il fatto che è abbastanza soggettivo, è contrario alle regole dell'italiano e crea un pericoloso precedente, che potrebbe motivare altri innumerevoli e minuscoli cambiamenti simili.
Per questi motivi ti direi che è molto molto meglio evitare, ma se proprio non ti va giù un compromesso meno vistoso potrebbe essere invertirli in "Solo Bluetooth o auricolari"